FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, from: response))
+
+			default:
+				completion(self.failure(.connectivity))
+			}
+		}
+	}
+
+	private func failure(_ error: RemoteFeedLoader.Error) -> FeedLoader.Result {
+		return .failure(error)
+	}
+}
+
+private final class FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Image]
+
+		var feed: [FeedImage] {
+			return self.items.map { $0.image }
```

Comment:
Please remove `self` to follow the codebase convention

```suggestion
			return items.map { $0.image }
```